editoast/src/modelsv2/infra.rs

```diff
@@ -257,6 +262,30 @@ impl Infra {
             .await?;
         Ok(speed_limits_tags)
     }
+
+    pub async fn get_splited_track_section_with_data(
+        &self,
+        conn: &mut DbConnection,
+        track: Identifier,
+        distance_fraction: f64,
+    ) -> Result<Vec<SplitedTrackSectionWithData>> {
+        let query = format!("SELECT
+            object_table.obj_id as obj_id,
+            object_table.data as railjson,
+            ST_AsGeoJSON(ST_LineSubstring(ST_GeomFromGeoJSON(object_table.data->'geo'), 0, $3))::jsonb as left_geo,
+	        ST_AsGeoJSON(ST_LineSubstring(ST_GeomFromGeoJSON(object_table.data->'geo'), $3, 1))::jsonb as right_geo
+            FROM {} AS object_table
+            WHERE object_table.infra_id = $1 AND object_table.obj_id = $2",
+            get_table(&ObjectType::TrackSection),
+        );
```

Comment:
If we want to continue on this path of homogeneisation, then this SQL request can be put in a separate file.

Reply:
this one is a bit unusual, we have the name of the table which is a variable, and I don't know how to put it as a parameter in `sql_query`

Reply:
Yeah I recall trying to do that a long time ago and I believe that's impossible. Ideally to avoid that we'd rewrite the request using diesel's DSL, but since there's postgis functions and all that's not trivial.

However since the table is constant here I'd say it's fine to directly write `infra_object-tracksection`